backend/config/curation-api.yml

```diff
@@ -646,12 +646,13 @@ components:
           type: string
         processing_status:
           "$ref": "#/components/schemas/processing_status"
-          description: Processing status for Datasets in a Collection
+          description: >-
+            Processing status for Datasets in a Collection
 
-            - **"FAILURE":** one or more Datasets ran into an error during processing. The CELLxGENE team will reach out to help you.
-            - **"PENDING":** one or more Datasets are still being processed.
-            - **"SUCCESS":** all Datasets have been processed successfully.
-            - **null:** the Collection has no Datasets.
+            * `"FAILURE"`: one or more Datasets ran into an error during processing. The CELLxGENE team will reach out to help you.
+            * `"PENDING"`: one or more Datasets are still being processed.
+            * `"SUCCESS"`: all Datasets have been processed successfully.
+            * `null`: the Collection has no Datasets.
```

Comment:
just a stylistic change to improve accuracy/readability

Reply:
double check that it works. I kept it as it was because it was having trouble rendering, because of how it parses the `:`